### PR TITLE
Scc 221

### PR DESCRIPTION
**Pull request by cbrantley4**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function addEventListenersByView(st) {
     );
 
   // Add event listeners for the Form view
```
Comment (JohanBester) [inline index.js]: You removed the code for the 'form' in most places throughout the SPA, so this code for the form needs to be removed as well no?

File: index.js [hunk 2]
```diff
@@ -112,48 +112,52 @@ function fetchDataByView(done, st = state.Home) {
         });
       break;
     case "Blog":
```
Comment (JohanBester) [inline index.js]: The 'Blog' code has also mostly been gutted out of the SPA so this Blog code should go as well right?

**Review by JohanBester — COMMENTED** [review]
I have made some comments...

Not sure why we are pushing the Parcel-Cash to GitHub. Maybe that should be added to GitIgnore ?

I also made 2 comments on the root index.js.

If we are gutting out the code and example of the photo gallery and the form, then we should probably remove all the code related to those functions no?

**Review by JohanBester — APPROVED** [review]

File: .env [hunk 3]
```diff
@@ -1,3 +1,4 @@
-API_URL="http://localhost:4040"
-DB_CONNECT="mongodb://localhost/pizza"
-DB_PORT=4040
+OPEN_WEATHER_MAP_API_KEY=fbb30b5d6cf8e164ed522e5082b49064
```
Comment (matsinet) [inline .env]: Remove the `.env` file from the repo
Create a `.env.dist` with the appropriate keys and neutral values

**Review by matsinet — APPROVED** [review]
Looks good to me

**Pull request merged**